### ISSUE TITLE
Add minimum system requirements

### ISSUE THREAD

**Issue by ashabed** [geerlingguy/drupal-vm#454]
I spent a few days trying to get Drupal vm set up on a Windows 10 machine with an i3 core processor, when I eventually found that a required virtualbox setting was incompatible with the hardware. I'll try to add the specifics to this ticket a little later on.

Request: It would be helpful to document what the minimum hardware (and OS) requirements are for running DrupalVM.

I'm happy to submit a PR, if you are already aware of any limitations please let me know so that I can include them.

**Comment by geerlingguy** [comment]
@ashabed - Is there a setting that we could toggle to make Drupal VM work correctly on an i3? I don't have one to test, but I wonder if we switched `v.customize ["modifyvm", :id, "--ioapic", "on"]` or if I built a more custom base box, if it could work on an i3 or non-Intel processors...?

**Comment by ashabed** [comment]
I believe it failed because his processor did not support Virtualization Technology (VT-x). The option was grayed out in VirtualBox, and I read on stackoverflow that many i3's do not support it, as well as that it can cause the 'system entered an invalid state: poweroff' error. I'm not sure what change could be made to run DrupalVM without requiring VT-x. If there's something you'd like us to try I'd be happy to do so when I see him on Sunday. 

I asked him to provide me with the exact processor number so I can look up the specs, it seems some i3's do support VT-x.

**Comment by geerlingguy** [comment]
@ashabed - It looks like if VT-x is not available, 64-bit guests can't run on a 32-bit system. See: http://superuser.com/a/749369/80658

So, one way to bypass this restriction would be to use a 32-bit base box. I don't build any, but you could use a box from Atlas:

  - `hashicorp/precise32` (12.04)
  - `ubuntu/trusty32` (14.04)

**Comment by ashabed** [comment]
Thank you for this. I met with him yesterday, we were actually able to enable VT-x, so we did get this working on an i3.

@geerlingguy Before I looked up his specific processor and found that it actually does support VT-x, I had tried out the 2 boxes you listed above and was still receiving the same VT-x error as before. Just an FYI - I'm not sure those boxes would fix it for someone who does not have VT-x.
